part2/src/components/GradientButton.js

```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components";
+
+const SIZES = {
+  large: 350,
+  medium: 128,
+  small: 80,
+};
+
+const GradientButton = styled.button`
+  width: ${({ size }) => SIZES[size] ?? SIZES["medium"]}px;
```

Comment:
이 코드는 아래와 같이 수정하면 좀 더 간결하게 작성될 수 있을것 같습니다.
자바 스크립트의 병합 연산자는 많이 사용되는 연산자이기 때문에 사용법을 잘 익혀두시면 좋습니다.
잘 작성해 주셨지만 불필요한 코드가 없는지 한 번씩 고민해보시면 더 좋은 코드를 작성 하실 수 있을것 같습니다.
```suggestion
  width: ${({ size }) => SIZES[size ?? "medium"]}px;
```